source/partials/_speakers.html.erb

```diff
@@ -0,0 +1,96 @@
+<div class="speakers">
+  <h3 class="speakers__title">Speakers <span>.</span></h3>
+  <div class="speakers__separator"></div>
+
+  <div class="speakers__container">
+
+    <div class="speakers__speaker">
+      <div class="speakers__photo">
+        <div class="speakers__social">
+          <a href="#" class="speakers__social__media"> <%= partial 'partials/svg/logo-facebook-black' %></a>
+          <a href="#" class="speakers__social__media"> <%= partial 'partials/svg/logo-twitter-black' %></a>
+          <a href="#" class="speakers__social__media"> <%= partial 'partials/svg/logo-linkedin-black' %></a>
+          <a href="#" class="speakers__social__media"> <%= partial 'partials/svg/logo-instagram-black' %></a>
+        </div>
+
+        <figure>
+          <img src="http://evently.mikado-themes.com/wp-content/uploads/2017/06/design-team-img-2.jpg" alt="Ada Patterson">
+          <div class="speakers__photo__border"></div>
```

Comment:
In this case, Could you use a pseudo-element `:before` or `:after` to do this effect?

Reply:
Yes. I removed the div and added `::before`